File: backend/cache/redisClient.ts
import { createClient } from 'redis';
import { config } from '../config/config';
export const redisClient = createClient({
  url: config.REDIS_URL
});
redisClient.on('error', (err) => console.error('Redis Client Error', err));
redisClient.connect().catch(console.error);

File: backend/cache/redis_client.py
import redis
from ..config import Config
redis_client = redis.Redis.from_url(Config.REDIS_URL)
def redis_get(key):
    return redis_client.get(key)
def redis_set(key, value, ttl=300):
    redis_client.setex(key, ttl, value)
